src/addRemoveToDo.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable import/no-cycle */
+/* eslint-disable import/no-mutable-exports */
+
+import getTodoList from './index.js';
+
+export let storedList = JSON.parse(localStorage.getItem('tasks'));
+if (storedList === null) {
+  storedList = [];
+}
+
+export const add = (description) => {
+  const todolist = [...storedList];
+  todolist.push({
+    index: Date.now(),
+    description,
+    completed: false,
```

Comment:
when I mark a task as completed , the value of completed of a given task in the local storage should be updated to true , In your case its not updating, Kindly fix this issue 👍 👍